src/core_plugins/kibana/public/management/sections/settings/advanced_row.html

```diff
@@ -69,12 +69,12 @@
         ng-model="conf.unsavedValue"
         ng-options="option as option for option in conf.options"
         class="form-control"
-        data-test-subj="selectInput">
+        data-test-subj="advancedSetting {{conf.name}} selectInput">
```

Comment:
Changing the selectors used in the [functional tests](https://github.com/elastic/kibana/blob/master/test/support/page_objects/settings_page.js#L35) is an option too